ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/defs_node_type.schema.yml

```diff
@@ -63,6 +63,9 @@ $defs:
             description: Node management interface IPv4 address.
             type: str
             format: cidr
+          mgmt_gateway:
```

Comment:
Can we implement same for ipv6 aslo since ipv6_mgmt_ip support already there.
cc: @ClausHolbechArista @gmuloc

Reply:
Sure!

Reply:
Added with tests.